mne/gui/_fiducials_gui.py

```diff
@@ -168,20 +168,23 @@ def _subject_changed(self):
         if not subjects_dir or not subject:
             return
 
-        path = None
+        # find head model
         if self.use_high_res_head:
-            path = _find_high_res_head(subjects_dir=subjects_dir,
-                                       subject=subject)
+            path = _find_head_bem(subject, subjects_dir, high_res=True)
             if not path:
                 error(None, "No high resolution head model was found for "
                       "subject {0}, using standard head instead. In order to "
                       "generate a high resolution head model, run:\n\n"
                       "    $ mne make_scalp_surfaces -s {0}"
                       "\n\n".format(subject), "No High Resolution Head")
-
-        if not path:
-            path = head_bem_fname.format(subjects_dir=subjects_dir,
-                                         subject=subject)
+                path = _find_head_bem(subject, subjects_dir)
+        else:
+            path = _find_head_bem(subject, subjects_dir)
```

Comment:
path = _find_head_bem(subject, subjects_dir)

should be before if as you call it in 2 branches